roseau/load_flow/network.py

```diff
@@ -43,6 +46,96 @@
+            their IDs as keys. LV networks typically have one ground element connected to the
+            neutral of the main source bus (secondary of the MV/LV transformer). HV networks
+            may have one or more grounds connected to the shunt components of their lines.
+
+        potential_refs:
+            The potential references of the network. Either a list of potential references or a
+            dictionary of potential references with their IDs as keys. As the name suggests, this
+            element defines the reference of potentials of the network. A potential reference per
+            galvanically isolated section of the network is expected. A potential reference can
+            be connected to a bus or to a ground.
+
+    Attributes:
+        DEFAULT_PRECISION (float):
+            The default precision needed for the convergence of the load flow algorithm. The solver
+            stops when the error between two iterations is below this value. Default is 1e-6.
+
+        DEFAULT_MAX_ITERATIONS (int):
+            Maximum number of iterations to perform the load flow analysis. The solver stops when
+            this number of iterations is reached. Default is 20.
+
+        DEFAULT_BASE_URL (str):
+            Base URL of the Roseau Load Flow API endpoint.
+
+        buses (dict[Id, Bus]):
+            Dictionary of buses of the network indexed by their IDs. Also available as a
+            :attr:`GeoDataFrame<buses_frame>`.
+
+        branches (dict[Id, AbstractBranch]):
+            Dictionary of branches of the network indexed by their IDs. Also available as a
+            :attr:`GeoDataFrame<branches_frame>`.
+
+        loads (dict[Id, AbstractLoad]):
+            Dictionary of loads of the network indexed by their IDs. Also available as a
+            :attr:`DataFrame<loads_frame>`.
+
+        sources (dict[Id, VoltageSource]):
+            Dictionary of voltage sources of the network indexed by their IDs. Also available as a
+            :attr:`DataFrame<sources_frame>`.
+
+        grounds (dict[Id, Ground]):
+            Dictionary of grounds of the network indexed by their IDs. Also available as a
+            :attr:`DataFrame<grounds_frame>`.
+
+        potential_refs (dict[Id, PotentialRef]):
+            Dictionary of potential references of the network indexed by their IDs. Also available
+            as a :attr:`DataFrame<potential_refs_frame>`.
+
+        res_info (JsonDict):
+            Dictionary containing solver information on the last run of the load flow analysis.
+            Empty if the load flow analysis has not been run yet.
+            Example::
+
+                {
+                    "status": "success",
+                    "resolution_method": "newton",
+                    "iterations": 2,
+                    "target_error": 1e-06,
+                    "final_error": 1.9468870959826745e-12
+                }
```

Comment:
Now, there are the additional `warm_start` and `max_iterations` keys. The key `target_error` has been renamed `precision` and `final_error` was renamed `final_precision`